scripts/install_prereqs.sh

```diff
@@ -67,6 +67,11 @@ case "$PROJECT-$PLATFORM-$METHOD" in
     pip install six
     ;;
 
+  SymbolCollision-*-xcodebuild)
```

Comment:
I don't quite understand how this label works, but is it intended that SymbolCollision-xcodebuild-* is used in build.sh and SymbolCollision-*-xcodebuild is here?

Reply:
The case statements in the two scripts are inconsistent. We should fix, but out of scope for this PR.